### DISCUSSION TITLE
Why does `flutter build windows --release` automatically download the nightly version of rust?

### DISCUSSION THREAD

**Discussion by Jhonfunk** [fzyzcjy/flutter_rust_bridge#1673] [Q&A] [answered]
>> os: windows 11

In vscode the stable version of rust is used, but in the command line environment the nightly version of rust is automatically downloaded and used.

**Comment by fzyzcjy** [comment, marked as answer]
flutter_rust_bridge core allows arbitrary ways to integrate Flutter+Rust, and I guess you mean the default template, which uses Cargokit. Thus, the question may be, why cargokit downloads Rust nightly. Maybe firstly try `--verbose` and look at / paste the logs here, maybe cargokit logs will say something about this.

**Reply by Jhonfunk** [reply]
https://github.com/superlistapp/super_native_extensions/blob/main/super_native_extensions/rust/cargokit.yaml

Thanks for your help, it seemed inevitable.

**Reply by fzyzcjy** [reply]
Your yaml hints me to search in cargokit main repo https://github.com/irondash/cargokit and gets: https://github.com/irondash/cargokit/blob/f75677242424e19e5fcc6184278571280ac02fa0/docs/architecture.md?plain=1#L65

Maybe create a new cargokit.yaml and set "stable" instead of "nightly", I guess that may help
